app/components/ContentEditable.tsx

```diff
@@ -129,6 +129,29 @@ const ContentEditable = React.forwardRef(
       []
     );
 
+    if (React.Children.count(children) > 1) {
+      return (
+        <div className={className} dir={dir} onClick={onClick}>
+          {React.Children.toArray(children)[0]}
+          <Content
+            ref={contentRef}
+            contentEditable={!disabled && !readOnly}
+            onInput={wrappedEvent(onInput)}
+            onBlur={wrappedEvent(onBlur)}
+            onKeyDown={wrappedEvent(onKeyDown)}
+            onPaste={handlePaste}
+            data-placeholder={placeholder}
+            suppressContentEditableWarning
+            role="textbox"
+            {...rest}
+          >
+            {innerValue}
+          </Content>
+          {React.Children.toArray(children)[1]}
+        </div>
+      );
+    }
+
```

Comment:
Resembles a _point hack_ to _position_ the `EmojiPicker` in this component. Although, it works fine but I'm sure there are better ways. One way is to move the _first child_(referencing `EmojiPicker`) out and wrap it in `EditableTitle` but an initial trial there messed up its(emoji picker's) position in DOM. Inputs welcome! @tommoor